2.x-centos7/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER mail@racktear.com
 RUN groupadd tarantool \
     && adduser -g tarantool tarantool
 
-ENV TARANTOOL_VERSION=2.1.2-119-gc118d5142 \
+ENV TARANTOOL_VERSION=2.1.2-143-g3edaaed6c \
```

Comment:
Please, update this image to 2.2

Reply:
Updated 2.x-centos7 to the latest 2.2 version 2.2.1-3-g878e2a42c. Is it Ok ?

Reply:
Yes, thanks